src/main/java/hudson/plugins/jira/JiraGlobalConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package hudson.plugins.jira;
+
+import hudson.Extension;
+import hudson.util.PersistedList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import jenkins.model.GlobalConfiguration;
+import jenkins.model.Jenkins;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundSetter;
+
+@Extension
+@Symbol("jira")
+public class JiraGlobalConfiguration extends GlobalConfiguration {
+
+    @Nonnull
+    public static JiraGlobalConfiguration getInstance() {
```

Comment:
any reason why it's static?

Reply:
Cause a descriptor is usually in a static context, I cannot override get descriptor because I want to avoid calling `Jenkins.getInstance().getDescriptorOrDie(JiraGlobalConfiguration.class)` all over the place and I should not instantiate a descriptor, Jenkins does that job 😅